packages/logger/src/index.ts

```diff
@@ -12,10 +13,10 @@ const createLoggerInstance = (defaultMeta: { [key: string]: string; } = {}) => c
     ...(ENV.logFormat === 'json' ? jsonFormat() : simpleFormat()),
   ),
   defaultMeta: {
+    ...defaultMeta,
     serviceContext: {
-      service: ENV.serviceName,
-      version: `${ENV.serviceVersion}-${ENV.nodeEnv}`,
-      ...defaultMeta,
```

Comment:
This was the issue - `defaultMeta` already has `serviceContext` & `createLoggerInstance` should be able to use either the name & version from env variables or the ones provided via `setup`